documentation/src/main/asciidoc/user_guide/persistence_rocksdb.adoc

```diff
@@ -97,6 +97,24 @@ Configuration cacheConfig = new ConfigurationBuilder().persistence()
 
 ----
 
+[NOTE]
+====
+{brandname} servers log warning messages at startup when you add configuration
+properties that are specific to RocksDB.
+
+For example, as in the preceding example, if you set `data.write_buffer_size`
+in your configuration, then the following message is written to logs:
+
+----
+WARN  [org.infinispan.configuration.parsing.XmlConfigHelper] ISPN000292:
+Unrecognized attribute 'data.write_buffer_size'. Please check your
+configuration. Ignoring!
+----
+
+{brandname} applies the RocksDB properties even though the log message
+indicates they do not take effect. You can ignore these `WARN` messages. 
+====
+
```

Comment:
Hmm this is something we could definitely improve, but good to have it documented!